src/dsymbol/conversion/second.d

```diff
@@ -298,6 +301,53 @@ body
 	}
 }
 
+unittest
```

Comment:
True. I like the declarative approach, it is more compact.
but the alternative is more flexible, more things are testable for potential bugs, for instance: symbol.kind, symbol.qualifier , symbol.isPointer ...etc.
Maybe we should see if we can make it the current system more flexible.